passes/elim_dead_code.cpp

```diff
@@ -187,6 +187,20 @@ const IR::Node *ElimDeadCode::preorder(IR::MethodCallStatement *stmt) {
         }
     }
 
+    // Action annotated with `@defaultonly` is ignored before. If no action can be reachable
+    // based on previous analysis, we replace the apply call to the default action call.
+    // Only when the default action has an empty body, we remove the apply.
+    auto *defaultAction = table.getDefaultAction()->checkedTo<IR::MethodCallExpression>();
+    if (defaultAction != nullptr) {
+        auto *decl = refMap.get()
```

Comment:
Does `getActionDecl` not work here?

Reply:
Didn't notice this function before. Fixed.